### PR TITLE
Fix badges

### PR DESCRIPTION
**Pull request by hirosassa**
## What is this PR?

- added a badge shows supported python versions
- changed build status badge from travis' to github actions' one

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,4 +114,4 @@ class SomeTask(gokart.TaskOnKart):
 This is useful when multiple tasks has same parameter, since parameter settings of `MasterConfig`  will be inherited to all tasks decorated with `@inherits_config_params(MasterConfig)`.
 
 Note that parameters which exists in both `MasterConfig` and `SomeTask` will be inherited.
-In the above example, `param2` will not be available in `SomeTask`, since `SomeTask` does not have `param2` parameter.
\ No newline at end of file
+In the above example, `param2` will not be available in `SomeTask`, since `SomeTask` does not have `param2` parameter.
```
Comment (hirosassa) [inline README.md:117]: I didn't edit this line 🤔

File: README.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # gokart
 
-[![Build Status](https://travis-ci.org/m3dev/gokart.svg)](https://travis-ci.org/m3dev/gokart)
+[![Test](https://github.com/m3dev/gokart/workflows/Test/badge.svg)](https://github.com/m3dev/gokart/actions?query=workflow%3ATest)
 [![](https://readthedocs.org/projects/gokart/badge/?version=latest)](https://gokart.readthedocs.io/en/latest/)
+[![Python Versions](https://img.shields.io/pypi/pyversions/gokart.svg)](https://pypi.org/project/gokart/)
```
Comment (hirosassa) [inline README.md:5]: We need wait merging this PR for boto's release for python 3.9

**Comment by hirosassa** [comment]
This is how we look

Screen Shot 2021-01-31 at 6 55 47

File: README.md (hunk 2 above)
Comment (Hi-king) [reply README.md:5]: :)

**Review by Hi-king — APPROVED** [review]
LGTM! Can't wait for 3.9 compatibility

**Comment by vaaaaanquish** [comment]
:Cool:

**Review by vaaaaanquish — APPROVED** [review]

**Comment by hirosassa** [comment]
I'll rebase master and apply test on py39

**Comment by Hi-king** [comment]
@hirosassa plz fix this 🙇 
image

**Pull request closed without merging**